source/installation.rst

```diff
@@ -23,8 +23,91 @@ not present, contact PCDS and we will include it a subsequent release.
 
 .. note::
 
-   Do not create your own environments here. See the instructions below to
-   create your own environment if you want to try different packages
+   Do not create your own environments here.
+   See :ref:`ref-create-env` if you want to try different packages
+
+Using In-Development Packages
+=============================
+It is possible to develop new packages without creating your own environments.
```

Comment:
I might reword this and add a little bit of motivation.

"""
It is possible to include in-development packages without creating your own environment. This is useful if you made made changes to a library and want to see how it interacts with the rest of the released ecosystem. 
"""